bundles/mapping/drawtools/plugin/DrawPlugin.ol3.js

```diff
@@ -584,6 +665,11 @@ Oskari.clazz.define(
             var optionsForDrawingEvent = {
                 isFinished: false
             };
+
+            if (options.allowMultipleDrawing === 'multiGeom') {
+                me.drawMultiGeom = true;

```

Comment:
Nothing resets me.drawMultiGeom so after it has been set true it will remain so until the page is reloaded.